crates/rspack_plugin_javascript/src/ast/stringify.rs

```diff
@@ -32,7 +32,7 @@ pub fn stringify(
       EsVersion::Es2022,
       SourceMapConfig {
         enable: devtool.source_map(),
-        inline_sources_content: !devtool.no_sources(),
+        inline_sources_content: true,
```

Comment:
Setting this as the original one will result in a panic.
```
called `Option::unwrap()` on a `None` value.
```
Have to take a look afterwards.
Case: configCases › source-map › nosources › nosources should compile